### PR TITLE
Feat/met 1988 perform tier calculation

### PR DESCRIPTION
**Pull request by jochen-vermeulen**

### CODE REVIEW

File: metis-indexing/src/main/java/eu/europeana/indexing/utils/RdfTier.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import eu.europeana.indexing.tiers.model.MediaTier;
+import eu.europeana.indexing.tiers.model.MetadataTier;
+import eu.europeana.indexing.tiers.model.Tier;
+
+/**
+ * This enum lists the content and metadata tiers that exist.
+ */
+public enum RdfTier {
+
+  METADATA_TIER_0("http://www.europeana.eu/schemas/epf/metadataTier0", MetadataTier.T0),
```
Comment (stzanakis) [inline metis-indexing/src/main/java/eu/europeana/indexing/utils/RdfTier.java]: I believe the last part of the url should start with "#" and not a "/"?

File: metis-indexing/src/main/java/eu/europeana/indexing/tiers/model/MetadataTier.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package eu.europeana.indexing.solr.tiers.model;
+package eu.europeana.indexing.tiers.model;
 
 /**
  * This enum contains the different values of the metadata tier.
```
Comment (stzanakis) [inline metis-indexing/src/main/java/eu/europeana/indexing/tiers/model/MetadataTier.java]: Why is the label field required?

File: metis-indexing/src/main/java/eu/europeana/indexing/tiers/model/MediaTier.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package eu.europeana.indexing.solr.tiers.model;
+package eu.europeana.indexing.tiers.model;
 
 /**
  * This enum contains the different values of the media tier.
```
Comment (stzanakis) [inline metis-indexing/src/main/java/eu/europeana/indexing/tiers/model/MediaTier.java]: Why is the label field required?

File: metis-indexing/src/main/java/eu/europeana/indexing/utils/RdfTier.java (hunk 1 above)
Comment (jochen-vermeulen) [reply metis-indexing/src/main/java/eu/europeana/indexing/utils/RdfTier.java]: Good catch! However, I don't know. This document has them with slash:
https://docs.google.com/document/d/1BXdrIbyh2ufiC4WvDvisX8v1M89pAEVSne_tWXpHRdo/edit
And so does the description of this ticket:
https://europeana.atlassian.net/browse/MET-2008
However, the following ticket has it with hash (#):
https://europeana.atlassian.net/browse/MET-1987
Finally, this ticket has both versions (the table vs. the code snippet): 
https://europeana.atlassian.net/browse/MET-1988
So what is the right answer? I'm asking Valentine.

File: metis-indexing/src/main/java/eu/europeana/indexing/tiers/model/MetadataTier.java (hunk 2 above)
Comment (jochen-vermeulen) [reply metis-indexing/src/main/java/eu/europeana/indexing/tiers/model/MetadataTier.java]: I anticipated their use, but I saw I now have them also in QualityAnnotationSolrCreator, where they belong. I will remove them.

File: metis-indexing/src/main/java/eu/europeana/indexing/tiers/model/MediaTier.java (hunk 3 above)
Comment (jochen-vermeulen) [reply metis-indexing/src/main/java/eu/europeana/indexing/tiers/model/MediaTier.java]: I anticipated their use, but I saw I now have them also in QualityAnnotationSolrCreator, where they belong. I will remove them.

File: metis-indexing/src/main/java/eu/europeana/indexing/utils/RdfTier.java (hunk 1 above)
Comment (jochen-vermeulen) [reply metis-indexing/src/main/java/eu/europeana/indexing/utils/RdfTier.java]: As you can see in the ticket, the way I had it is correct.

**Comment by stzanakis** [comment]
It has been merged in develop but because of conflict it doesn't show up as merged

**Pull request closed without merging**